R/list-combine.R

```diff
@@ -58,19 +63,22 @@ list_cbind <- function(
     x,
     ...,
     name_repair = c("unique", "universal", "check_unique"),
-    size = NULL
+    size = NULL,
+    keep_empty = FALSE
   ) {
   check_list_of_data_frames(x)
   check_dots_empty()
+  if(keep_empty) x <- convert_empty_element_to_NA(x)
```

Comment:
Are you sure this applies to `list_cbind()`? I don't know why but I think it shouldn't?

Reply:
To me, it's unclear what type the empty column should have.

Reply:
It's also weird that for `list_cbind()` the name of empty columns comes from the outer name rather than the inner name.